### DISCUSSION TITLE
Failed to parse source map, no such file or directory

### DISCUSSION THREAD

**Discussion by alvisjosh** [mkosir/react-parallax-tilt#46] [General]
Hi there,

I have installed the library using: `npm i react-parallax-tilt`

A simply import and use of <Tilt> gives me a few warnings, one of them is as below:

`WARNING in ./node_modules/react-parallax-tilt/dist/index.esm.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/stevenkik/Documents/battle-era/Web/battleera/node_modules/react-parallax-tilt/src/features/glare/Glare.ts' file: Error: ENOENT: no such file or directory, open '/Users/stevenkik/Documents/battle-era/Web/battleera/node_modules/react-parallax-tilt/src/features/glare/Glare.ts'`

How to resolve those warning?

--
Alvis

**Comment by mkosir** [comment]
Hi @alvisjosh,
thx for reporting it, warnings should be fixed with newest version `v.1.7.9`.

I had troubles reproducing it, since in majority cases I'm using NextJs, but if you encounter same warnings with other packages, the reason is in [latest CRA v5.x](https://github.com/facebook/create-react-app/discussions/11767) (Webpack 5), where warnings can be disabled by setting env var `GENERATE_SOURCEMAP` to `false`.

**Reply by alvisjosh** [reply]
Great! My version was 1.7.8 with the reported warning.

Did an update just now and the warning is gone now. Thank you!

**Comment by KyawZinMyo7** [comment]
Hi ,

I have installed the library using: npm i react-parallax-tilt.

I got npm errs .

npm ERR! code ENOTEMPTY
npm ERR! syscall rename
npm ERR! path /Users/kyawzinmyo/facerecoginitionbrain/node_modules/acorn-globals
npm ERR! dest /Users/kyawzinmyo/facerecoginitionbrain/node_modules/.acorn-globals-IQau6AKW
npm ERR! errno -66
npm ERR! ENOTEMPTY: directory not empty, rename '/Users/kyawzinmyo/facerecoginitionbrain/node_modules/acorn-globals' -> '/Users/kyawzinmyo/facerecoginitionbrain/node_modules/.acorn-globals-IQau6AKW'

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/kyawzinmyo/.npm/_logs/2022-11-27T16_52_59_344Z-debug-0.log

how to solve this errs.Please.thankyou
